requirements.txt

```
@@ -24,4 +24,5 @@ marshmallow-enum>=1.5.1
 Pillow>=6
 sncosmo>=2.1.0
 tdtax==0.1.0
```

You can bump `tdtax` here to get a compatible pytest version.
```suggestion
tdtax>=0.1.1
```